go/vt/vtgate/planbuilder/ordered_aggregate.go

```diff
@@ -333,6 +335,18 @@ func (oa *orderedAggregate) Wireup(plan logicalPlan, jt *jointab) error {
 }
 
 func (oa *orderedAggregate) WireupGen4(semTable *semantics.SemTable) error {
+	colls := map[int]collations.ID{}
+	oa.eaggr.Collations = colls
```

Comment:
Why not doing directly:
```go
oa.eaggr.Collations = map[int]collations.ID{}
```

Reply:
I keep using this variable :point_down:, and I didn't want to spell out `oa.eaggr.Collations` every time